consensus/state_sync/module.go

```diff
@@ -1,18 +1,25 @@
 package state_sync
 
 import (
+	typesCons "github.com/pokt-network/pocket/consensus/types"
 	"github.com/pokt-network/pocket/logger"
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
 	"github.com/pokt-network/pocket/shared/modules"
 )
 
 const (
-	stateSyncModuleName = "stateSyncModule"
+	stateSyncModuleName       = "stateSyncModule"
+	committedBlocsChannelSize = 100
 )
 
 type StateSyncModule interface {
 	modules.Module
 	StateSyncServerModule
+
+	Set(aggregatedMetaData *typesCons.StateSyncMetadataResponse)
```

Comment:
You're specifying the type and name here but only the type of the variable below. It's inconsistent so please fix.